settings.py

```diff
@@ -282,3 +290,53 @@
 MME_SLACK_ALERT_NOTIFICATION_CHANNEL = 'matchmaker_alerts'
 MME_SLACK_MATCH_NOTIFICATION_CHANNEL = 'matchmaker_matches'
 MME_SLACK_SEQR_MATCH_NOTIFICATION_CHANNEL = 'matchmaker_seqr_match'
+
+#########################################################
+#  AnVIL Terra API specific settings
+#########################################################
+
+GOOGLE_AUTH_CONFIG_DIR = os.environ.get('GOOGLE_AUTH_CONFIG_DIR', '')
+
+GOOGLE_AUTH_CLIENT_CONFIG = {}
+GOOGLE_SERVICE_ACCOUNT_INFO = {}
+if GOOGLE_AUTH_CONFIG_DIR:
+    with open(os.path.join(GOOGLE_AUTH_CONFIG_DIR, 'client_secret.json'), 'r') as f:
+        GOOGLE_AUTH_CLIENT_CONFIG = json.load(f)
+    with open(os.path.join(GOOGLE_AUTH_CONFIG_DIR, 'service_account.json'), 'r') as f:
+        GOOGLE_SERVICE_ACCOUNT_INFO = json.load(f)
+
+
+#########################################################
+#  Social auth specific settings
+#########################################################
+
```

Comment:
should any of this be set up if we don't have `TERRA_API_ROOT_URL` configured?

Reply:
We still keep modules for Google-auth even when `TERRA_API_ROOT_URL` is not configured. These modules might use the settings below. I may take out any codes that could cause exceptions.

Reply:
but we don't include social auth as an installed app when there is no `TERRA_API_ROOT_URL` so what modules could possibly use these settings?

Reply:
Okay. I can make `social_auth` to be an optional app.